.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
```

Comment:
Shot in the dark but looks like this was deprecated

https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure